### PR TITLE
Fixes #165 Creates a simple starter UI for Sensors

### LINKED ISSUE #165: Create a generic application for reading data from I2C sensors
Field tests are necessary to test if the communication libraries for the sensors are working. 
it's easy to miss things while porting code, so  I propose to add a minimal UI which can auto-scan the I2C bus, let the user select a sensor, and fetch the data returned by the getRaw function.

A simple UI to quickly verify all the ported libraries should have these essentials.
+ autoscan (button) onClick -> display detected addresses
+ address (hexadecimal input / combobox with results of autoscan)
+ init (button) onClick -> Create an instance of the class associated with the selected address.
+ getRaw (button) onClick -> display array of data returned by the getRaw function

If we have the resources, we can make it slightly more sophisticated by adding the sensor-specific-functions in the form of widgets. But this might take a while, and is not critical for a minimal testing tool.
+ read the params HashMap, and create corresponding widgets
+ Associate callbacks to the corresponding functions in the sensor's class.

### PR DESCRIPTION
**Pull request by asitava1998**
Fixes issue #165 

Changes: 
* Creates a simple UI for Sensors.
* There is an auto scan button to detect sensors, the sensors detected will be displayed in the text view below & listed in the spinner.
* The sensor is selected from the spinner and then the get raw button is pressed to fetch raw data from the sensors.
* Detailed description of purpose and features is presented in issue description.

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/ReportUs.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_report_us);
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/activity/ReportUs.java]: Please change this back.
Probably a mistake

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -84,4 +84,10 @@
     <string name="digital_outputs">Digital Outputs</string>
     <string name="configure_PWM">Configure PWM</string>
     <string name="waveform_generator">Arbitrary Waveform Generator</string>
+    <string name="autoscan">AUTOSCAN</string>
+    <string name="select_sensor">SELECT SENSOR</string>
+    <string name="get_raw">GET RAW</string>
+    <string-array name="sensors">
+
```
Comment (viveksb007) [inline app/src/main/res/values/strings.xml]: sensor would be scanned on the fly. So why store an empty string array here?

File: app/src/main/res/layout/activity_sensor.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp">
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text=""
+                android:background="@drawable/control_custom_border"/>
+        </ScrollView>
+    </LinearLayout>
+</ScrollView>
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_sensor.xml]: Hello @asitava1998 this fix is really nice! When you're changing the things Vivek has pointed out, can you add a new line here? :blush:

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -84,4 +84,10 @@
...
+    </string-array>
 </resources>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: and here as well!

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -84,4 +84,10 @@
     <string name="digital_outputs">Digital Outputs</string>
     <string name="configure_PWM">Configure PWM</string>
     <string name="waveform_generator">Arbitrary Waveform Generator</string>
+    <string name="autoscan">AUTOSCAN</string>
+    <string name="select_sensor">SELECT SENSOR</string>
+    <string name="get_raw">GET RAW</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: We can undo these strings to be in normal case and set attributes in buttons to view them as uppercase! What do you think? :)

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (asitava1998) [inline app/src/main/res/values/strings.xml]: There is a layout rendering error if I don't reference a string array in the spinner.

File: app/src/main/res/layout/activity_sensor.xml (hunk 3 above)
Comment (asitava1998) [inline app/src/main/res/layout/activity_sensor.xml]: Okay. sure

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (asitava1998) [inline app/src/main/res/values/strings.xml]: :+1:

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (asitava1998) [inline app/src/main/res/values/strings.xml]: :+1:

File: app/src/main/java/org/fossasia/pslab/activity/ReportUs.java (hunk 1 above)
Comment (asitava1998) [inline app/src/main/java/org/fossasia/pslab/activity/ReportUs.java]: Yes. might have done some copy pasting in the wrong file accidently. I will correct it.

File: app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+public class SensorActivity extends AppCompatActivity {
+    private ScienceLab scienceLab;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_sensor);
+        scienceLab = ScienceLabCommon.getInstance().scienceLab;
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: now that sciencelab is static you don't need to get instance.
```sciencelab = ScienceLabCommon.scienceLab``` would do the job.

Comment (asitava1998) [inline app/src/main/java/org/fossasia/pslab/activity/SensorActivity.java]: I will correct in the next PR regarding the sensor UI where functions will be added

**Pull request merged**